cs-rin-ru-enhanced-mod.user.js

```diff
@@ -511,15 +528,39 @@ function dynamicFunction(data) {
 
 // FUNCTIONS
 function mentionify() {
-    if ($(".postbody").length > 0 && URLContains("viewtopic.php") && options.mentioning >= 1 && !document.querySelector('a[href^="./posting.php?mode=reply"] img').alt.includes('locked')) {
-        const replyLink = $("[title='Reply to topic']").parent().attr("href");
-        $(".gensmall div+ div:not(:has([title='Reply with mentioning']))").each(function () {
-            const postElem = $(this).parents().eq(7);
-            const postID = $(postElem).find("a[name]").last().attr("name").slice(1);
-            const author = $(postElem).find(".postauthor").text();
-            const authorID = $(postElem).find("[title=Profile]").parent().attr("href").split("u=")[1];
-            $(this).append("<a href='" + replyLink + "&do=mention&p=" + postID + "&u=" + authorID + "&a=" + encodeURIComponent(author) + "'><img src='https://i.imgur.com/uTA0dBI.png' alt='Reply with mentioning' title='Reply with mentioning'></a>");
-        });
+    if ($(".postbody").length > 0 && URLContains("viewtopic.php") && options.mentioning >= 1 && document.querySelector('a[href^="./posting.php?mode=reply"] img')) {
+        if (!document.querySelector('a[href^="./posting.php?mode=reply"] img').alt.includes('locked')) {
+            const replyLink = $("[title='Reply to topic']").parent().attr("href");
+            $(".gensmall div+ div:not(:has([title='Reply with mentioning']))").each(function () {
+                const postElem = $(this).parents().eq(7);
+                const postID = $(postElem).find("a[name]").last().attr("name").slice(1);
+                const author = $(postElem).find(".postauthor").text();
+                const authorID = $(postElem).find("[title=Profile]").parent().attr("href").split("u=")[1];
+                if (!quickReplyPanel) {
+                    $(this).append("<a href='" + replyLink + "&do=mention&p=" + postID + "&u=" + authorID + "&a=" + encodeURIComponent(author) + "'><img src='https://i.imgur.com/uTA0dBI.png' alt='Reply with mentioning' title='Reply with mentioning'></a>");
+                } else {
+                    $(this).append("<a href='javascript:void(0);'><img src='https://i.imgur.com/uTA0dBI.png' alt='Reply with mentioning' title='Reply with mentioning'></a>");
```

Comment:
Maybe it would be better to also put new images in repo and link to them on github?